packages/components/filter-bar/filter-bar.types.ts

```diff
@@ -104,7 +104,15 @@ export interface KbqSaveFilterError {
     nameAlreadyExists?: boolean;
 }
 
+/** saving statuses for KbqSaveFilterEvent */
+export enum KbqSaveFilterStatuses {
```

Comment:
так как будто проще читается, некритично
```suggestion
export enum KbqSaveFilterStatus {
```

Reply:
для меня enum это множественное число: `KbqAlertColors, KbqButtonStyles, KbqComponentColors, KbqBadgeColors и т.д.` .

Оно ведь даже при обращении читается как из множества `KbqSaveFilterStatuses` берешь более конкретный статус `status: KbqSaveFilterStatuses.OnlyChanges` и когда объявляешь ты указываешь множество статусов `status: KbqSaveFilterStatuses;` ну я думаю ты понял :)

Вообще интересно, что приходим к обсуждению подобных вещей :)

Reply:
я просто не нашел в интернете четкого правила, поэтому обсудить решил